ibridges/meta.py

```diff
@@ -451,15 +466,16 @@ def __repr__(self) -> str:
 
     def __str__(self) -> str:
         """User readable representation of MetaDataItem."""
-        return f"(key: {self.key}, value: {self.value}, units: {self.units})"
+        units = "" if self.units is None else self.units
+        return f"(key: '{self.key}', value: '{self.value}', units: '{units}')"
```

Comment:
This can be reverted to the old situation.
```suggestion
        return f"(key: '{self.key}', value: '{self.value}', units: '{self.units}')"
```